### ISSUE TITLE
Problem entering hours and minute manually

### ISSUE THREAD

**Issue by Sevven45** [danielmoncada/date-time-picker#57]
When I try to enter hours or minutes manually it automatically add a 0. I have to carefully select the number to replace it, so I have to do this twice for each number if I want to enter a number bigger than 9.

I was thinking maybe there is an incompatibility with something? I was able to unblock it by passing the maxlength of the input at 3 in the console if this informations can help.

Thanks in advance

**Comment by danielmoncada** [comment]
@Sevven45 thanks for the response. Another issue has been submitted for this as well, so I'm going to merge this with it. Closing to merge with https://github.com/danielmoncada/date-time-picker/issues/60
